cmd/identities/helpers_test.go

```diff
@@ -0,0 +1,73 @@
+package identities_test
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"testing"
+
+	"github.com/ory/x/cmdx"
+
+	"github.com/pkg/errors"
+
+	"github.com/ory/kratos/identity"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ory/kratos/cmd/cliclient"
+	"github.com/ory/kratos/driver"
+	"github.com/ory/kratos/driver/config"
+	"github.com/ory/kratos/internal"
+	"github.com/ory/kratos/internal/testhelpers"
+)
+
+func setup(t *testing.T, cmd *cobra.Command) driver.Registry {
+	conf, reg := internal.NewFastRegistryWithMocks(t)
+	_, admin := testhelpers.NewKratosServerWithCSRF(t, reg)
+	conf.MustSet(config.ViperKeyDefaultIdentitySchemaURL, "file://./stubs/identity.schema.json")
+	// setup command
+	cliclient.RegisterClientFlags(cmd.Flags())
+	cmdx.RegisterFormatFlags(cmd.Flags())
+	require.NoError(t, cmd.Flags().Set(cliclient.FlagEndpoint, admin.URL))
+	require.NoError(t, cmd.Flags().Set(cmdx.FlagFormat, string(cmdx.FormatJSON)))
+	return reg
+}
+
+func exec(cmd *cobra.Command, stdIn io.Reader, args ...string) (string, string, error) {
+	stdOut, stdErr := &bytes.Buffer{}, &bytes.Buffer{}
+	cmd.SetErr(stdErr)
+	cmd.SetOut(stdOut)
+	cmd.SetIn(stdIn)
+	defer cmd.SetIn(nil)
+	if args == nil {
+		args = []string{}
+	}
+	cmd.SetArgs(args)
+	err := cmd.Execute()
+	return stdOut.String(), stdErr.String(), err
+}
+
+func execNoErr(t *testing.T, cmd *cobra.Command, args ...string) string {
+	stdOut, stdErr, err := exec(cmd, nil, args...)
+	require.NoError(t, err)
+	require.Len(t, stdErr, 0, stdOut)
+	return stdOut
+}
+
+func execErr(t *testing.T, cmd *cobra.Command, args ...string) string {
+	stdOut, stdErr, err := exec(cmd, nil, args...)
+	require.True(t, errors.Is(err, cmdx.ErrNoPrintButFail))
+	require.Len(t, stdOut, 0, stdErr)
+	return stdErr
+}
```

Comment:
This part is now part of `cmdx`, maybe remove it in this go? I can also do that btw

Reply:
Ah yeah makes sense, I first have to wrap everything in new* funcs to make it pass. We can then do another pass and remove these helpers and move to ory/x. Would be great if you could do that. I think it makes sense to do it once this PR is merged

Reply:
They are already in `ory/x` 1:1, you can just remove them here and import instead. But we can also separate that.

Reply:
The method signatures are a bit different so I'd separate it